updater.py

```diff
@@ -90,6 +93,29 @@ def do_update(args):
     current_link.symlink_to(release_dir, target_is_directory=True)
     print('Deployed', release_name)
 
+    do_cleanup(releases_dir, release_name)
+
+
+def do_cleanup(releases_dir, latest_release):
+    files = os.listdir(releases_dir)
+    removed = 0
+
+    if len(files) <= HISTORY_COUNT:
+        print('Did not run clean-up (too few historic releases)')
+        return
+
+    files.sort(reverse=True)
+
```

Comment:
why?

Reply:
oh, are the release dirs named ordinally, so this is used to skip the `HISTORY_COUNT` most recent?

Reply:
```
>>> releases = ['v1.1.1', 'v1.2.2', 'v1.2.1']
>>> releases.sort(reverse=True)
>>> print(releases)
['v1.2.2', 'v1.2.1', 'v1.1.1']
```

Most recent releases first -> anything after the nth list item is deleted.

Reply:
ok, so this will fail once you have more than 9 major, minor, or patch releases:

```
>>> a = ['1.2.3', '1.2.30', '1.2.15', '1.2.20', '1.2.2']
>>> a.sort(reverse=True)
>>> a
['1.2.30', '1.2.3', '1.2.20', '1.2.2', '1.2.15']
```

I've been bitten by this bug before :smile: